### ISSUE TITLE
Add privacy policy

### ISSUE THREAD

**Issue by rubenwardy** [minetest/minetest.github.io#135]
To apply to the forums, contentdb, wikis, and other sites

**Comment by Ezhh** [comment]
Agreed - this is very much needed. Will be more than happy to review and to help whoever takes it on.

**Comment by b3u** [comment]
I'm doing this using termly.io

**Comment by b3u** [comment]
How's this? https://app.termly.io/document/privacy-policy/3c7efca5-2832-487a-a229-77500e394923

**Comment by sofar** [comment]
email should be a minetest.net address.

**Comment by Ezhh** [comment]
What has this been taken from? We don't do things like marketing emails and Minetest isn't a business, so even from a quick skim-read, there are chunks that aren't relevant. 

The first step would actually be a little exercise finding out exactly what information is collected and where it is collected from, along with identifying exactly what is done with it (I'd imagine not much, and it just being basic info from the website and what people enter when registering for a forum account).

That can then be explained in simple english without any need for fancy stuff, along with a disclaimer that we will comply with any lawful requests regarding the data we hold, including, where relevant, replying to access/deletion/other requests that are a data subject's rights under the GDPR or other applicable legislation.  (From what I recall, everything is hosted in Europe, so I'm not sure if we'd need to specifically mention anything else - but I'm no lawyer, so feel free to correct me.)

Either way, the ones managing those websites would need to give input and being involved in this, and it should be kept as clean and simple as possible.
